windows/CodePush.Net46/FileUtils.cs

```diff
@@ -0,0 +1,62 @@
+﻿using PCLStorage;
+using System;
+using System.IO;
+using System.Threading.Tasks;
+
+namespace CodePush.ReactNative
+{
+    internal class FileUtils
+    {
+        internal async static Task MergeFoldersAsync(IFolder source, IFolder target)
+        {
+            foreach (IFile sourceFile in await source.GetFilesAsync().ConfigureAwait(false))
+            {
+                await CopyFileAsync(sourceFile.Path, Path.Combine(target.Path, sourceFile.Name)).ConfigureAwait(false);
+            }
+
+            foreach (IFolder sourceDirectory in await source.GetFoldersAsync().ConfigureAwait(false))
+            {
+                var nextTargetSubDir = await target.CreateFolderAsync(sourceDirectory.Name, CreationCollisionOption.OpenIfExists).ConfigureAwait(false);
+                await MergeFoldersAsync(sourceDirectory, nextTargetSubDir).ConfigureAwait(false);
+            }
+        }
+        internal async static Task ClearReactDevBundleCacheAsync()
+        {
+            try
+            {
+
+                var devBundleCacheFile = await FileSystem.Current.LocalStorage.GetFileAsync(CodePushConstants.ReactDevBundleCacheFileName).ConfigureAwait(false);
+
+                if (devBundleCacheFile != null)
+                {
+                    await devBundleCacheFile.DeleteAsync().ConfigureAwait(false);
+                }
+            }
+            catch (FileNotFoundException)
+            {
+                //no files do nothing
+            }
+        }
+
+        internal static Task<long> GetBinaryResourcesModifiedTimeAsync(string fileName)
+        {
+            var pathToAssembly = CodePushUtils.GetAppFolder();
+            var pathToAssemblyResource = Path.Combine(pathToAssembly, CodePushConstants.AssetsBundlePrefix.Replace("ms-appx:///", String.Empty), fileName);
```

Comment:
> CodePushConstants.AssetsBundlePrefix.Replace("ms-appx:///", String.Empty) [](start = 69, length = 74)

In react-native-windows, we eliminated the use of `ms-appx:///` in the context of react-native-wpf because the path prefix is controlled by the application.  Since ms-appx:/// is a UWP convention, it should not be used here.